packages/venia-concept/src/index.js

```diff
@@ -62,3 +62,8 @@ window.addEventListener('online', () => {
 window.addEventListener('offline', () => {
     store.dispatch(app.setOffline());
 });
+
+if (module.hot) {
+    // When any of the dependencies to this entry file change we should hot reload.
+    module.hot.accept();
```

Comment:
This simply makes it so the page doesn't hard refresh on every change.

Reply:
Just wanted to document from our chat in the office; this code will not make it into the production bundle, given module.hot will not be loaded.